src/screens/MapViewScreen.js

```diff
@@ -78,7 +76,7 @@ const styles = StyleSheet.create({
   augmentedRealityInfoContainer: {
     width: '90%'
   },
-  map: {
+  mapStyle: {
```

Comment:
no! why changing it? now it is different to the new sue map view screen. this is not related to any topic of the PR. please keep the file as it was.

Reply:
I need to update this to `mapStyle`. Otherwise it will not match `styles.mapStyle` in line 43. Or I need to update the style in line 43 to `styles.map`.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1140/commits/121e12118e6a32cde20cd6db6267c2a568c3d217